packages/commonwealth/client/scripts/views/components/Header/LoginSelectorMenuLeft.tsx

```diff
@@ -111,7 +111,7 @@ export const LoginSelectorMenuLeft = () => {
       >
         <CWText type="caption">Connect a new address</CWText>
       </div>
-      <Modal
+      <CWModal
```

Comment:
The `changeSelectedAddress` should be added to the dependency array. We can memoize `changeSelectedAddress`.